apps/mobile/src/screens/FundingRoundDetail/FundingRoundDetail.js

```diff
@@ -0,0 +1,605 @@
+import React, { useState, useEffect, useMemo } from 'react'
+import { View, Text, ScrollView, TouchableOpacity, Image, Alert, Animated, Platform } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { Check, DoorOpen } from 'lucide-react-native'
+import { useTranslation } from 'react-i18next'
+import { useNavigation } from '@react-navigation/native'
+import { TextHelpers } from '@hylo/shared'
+import useOpenURL from 'hooks/useOpenURL'
+import { personUrl } from '@hylo/navigation'
+import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
+import useFundingRound from '@hylo/hooks/useFundingRound'
+import useFundingRoundParticipation from '@hylo/hooks/useFundingRoundParticipation'
+import HyloHTML from 'components/HyloHTML'
+import HyloWebView from 'components/HyloWebView'
+import KeyboardFriendlyView from 'components/KeyboardFriendlyView'
+import Loading from 'components/Loading'
+import RoundPhaseStatus from 'components/RoundPhaseStatus'
+import SubmissionCard from 'components/SubmissionCard'
+import useRouteParams from 'hooks/useRouteParams'
+
+const TabButton = ({ isSelected, onPress, children }) => (
+  <TouchableOpacity
+    className={`py-1 px-4 rounded-md border-2 ${
+      isSelected
+        ? 'bg-selected border-selected'
+        : 'bg-transparent border-foreground/20'
+    }`}
+    onPress={onPress}
+    hitSlop={{ top: 4, bottom: 4, left: 6, right: 6 }}
+  >
+    <Text className='text-foreground'>{children}</Text>
+  </TouchableOpacity>
+)
+
+// Get current phase of funding round
+function getRoundPhase (round) {
+  if (!round) return 'draft'
+
+  const now = new Date()
+  const toDate = (value) => value ? new Date(value) : null
+
+  const submissionsOpenAt = toDate(round.submissionsOpenAt)
+  const submissionsCloseAt = toDate(round.submissionsCloseAt)
+  const votingOpensAt = toDate(round.votingOpensAt)
+  const votingClosesAt = toDate(round.votingClosesAt)
+
+  if (votingClosesAt && votingClosesAt <= now) return 'completed'
+  if (votingOpensAt && votingOpensAt <= now) return 'voting'
+  if (submissionsCloseAt && submissionsCloseAt <= now) return 'discussion'
+  if (submissionsOpenAt && submissionsOpenAt <= now) return 'submissions'
+  if (round.publishedAt && new Date(round.publishedAt) <= now) return 'open'
+
+  return 'draft'
+}
```

Comment:
If this something we can add to a FundingRound presenter, that can be maintained once and used on several clients?

Reply:
yes we could